modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -3,327 +3,317 @@ ifndef::env-site,env-github[]
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy to use for connectivity errors. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Data Query URL a| String |  The URL that is going to use to retrieve the entities |  #[payload] | 
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* non-repeatable-iterable |  Configure to use repeatable streams. |  | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
 |===
 
-=== Output
-
-[cols="50a,50a"]
+==== Output
+[%autowidth.spread]
 |===
-| Type | Array of Object
+|Type |Array of Object
 |===
 
-=== For Configurations
+==== For Configurations
+* <<dynamics-365-for-operations>> 
 
-* dynamics-365-for-operations
+==== Throws
 
-=== Throws
-
-* DYNAMICS365FOROPERATIONS:UNKNOWN
-* DYNAMICS365FOROPERATIONS:INVALID_CREDENTIALS
-* DYNAMICS365FOROPERATIONS:INVALID_CONNECTION
-* DYNAMICS365FOROPERATIONS:LOGIN_FAILED
-* DYNAMICS365FOROPERATIONS:CONNECTIVITY
+* DYNAMICS365FOROPERATIONS:CONNECTIVITY 
+* DYNAMICS365FOROPERATIONS:INVALID_CONNECTION 
+* DYNAMICS365FOROPERATIONS:INVALID_CREDENTIALS 
+* DYNAMICS365FOROPERATIONS:LOGIN_FAILED 
+* DYNAMICS365FOROPERATIONS:UNKNOWN 
 
 
 [[retrieveMultipleByQuery]]
-== Retrieve Multiple By Query
-
+=== Retrieve Multiple By Query
 `<dynamics365ForOperations:retrieve-multiple-by-query>`
 
-Retrieve Multiple entities by DSQL Query.
 
-=== Parameters
+Retrieve multiple entities by DSQL query.
+
 
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Datasense Query a| String |  The DSQL query that is going to be used for retrieve. The query transforms internally into a URL. |  `#[payload]` |
-| Streaming Strategy a| * repeatable-in-memory-iterable
-* repeatable-file-store-iterable
-* non-repeatable-iterable |  Configure to use repeatable streams and their behavior. |  |
-| Target Variable a| String |  The name of a variable that stores the output of the operation. |  |
-| Target Value a| String |  An expression that evaluates the operation's output and stores the outcome of that expression in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy to use for connectivity errors. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Datasense Query a| String |  The DSQL query that is going to use for retrieve. The query is transformed into a URL internally. |  #[payload] | 
```

Comment:
The DSQL query to use to retrieve entities...  ??  I don't know, but as is doesn't make sense.